llvm/lib/CodeGen/SelectionDAG/DAGCombiner.cpp

```diff
@@ -14818,15 +14818,14 @@ SDValue DAGCombiner::visitTRUNCATE(SDNode *N) {
     if (SDValue Reduced = reduceLoadWidth(N))
       return Reduced;
 
-    // Handle the case where the load remains an extending load even
+    // Handle the case where the load remains a normal/extending load even
     // after truncation.
```

Comment:
Nit: maybe rewrite this comment? "normal/extending load" seems to cover all relevant options so it's not really saying anything useful now as-is?

Maybe something like "Handle the case where the truncated result is at least as wide as the loaded type"